src/main/java/org/killbill/billing/payment/api/PaymentApi.java

```diff
@@ -646,4 +646,13 @@ public void setDefaultPaymentMethod(Account account, UUID paymentMethodId, Itera
     @RequiresPermissions(PAYMENT_METHOD_CAN_UPDATE)
     public List<PaymentMethod> refreshPaymentMethods(Account account, Iterable<PluginProperty> properties, CallContext context)
             throws PaymentApiException;
+
+    /**
+     * @param paymentTransactionId the payment transaction id
+     * @param context         the call context
+     * @return the payment transaction
+     * @throws PaymentApiException
+     */
+    public PaymentTransaction getPaymentTransactionById(final UUID paymentTransactionId, final TenantContext context)
+            throws PaymentApiException;
 }
```

Comment:
I thought [we said given a transaction, we would return the payment](https://github.com/killbill/killbill/issues/599#issuecomment-249255742):

```
Payment getPaymentByTransactionId(paymentTransactionId, bool, bool, properties, context);
```

Any reason why we should return the `PaymentTransaction` instead?